lib/sprockets/commoner/processor.rb

```diff
@@ -44,20 +47,14 @@ def self.call(input)
       end
 
       attr_reader :include, :exclude, :babel_exclude
-      def initialize(root, include: [root], exclude: [], babel_exclude: [/node_modules/])
+      def initialize(root, include: [root], exclude: ['vendor/bundle'], babel_exclude: [/node_modules/])
```

Comment:
Why does this folder need to be excluded? I wasn't clear on that from the discussion in the Slack channel.